.github/scripts/install-deps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
A lot of this feels like it should be in the documentation? IE Before getting started, make sure you have `git` and `wget` installed?

Reply:
This is available in the documentation: 
https://github.com/SymbiFlow/symbiflow-examples/pull/88/files#diff-f499103696248c617ff21a08e0a1a31cedab918185a5317a48c99c8997b22f54R14